### PR TITLE
adding related issue for a test failure

### PR DESCRIPTION
**Pull request by justnpT**

### CODE REVIEW

**Comment by Archbold** [comment]
@justnpT could you use more simpler English as I can guess but i don't really know what you mean by, " _the failure is not fully repetitive,_"

Also please add whether test should be tested manually or not.

**Comment by justnpT** [comment]
@Archbold  thank you for the CR fixed are in place

File: src/test/java/com/wikia/webdriver/testcases/forumtests/ForumBoardTests.java [hunk 1]
```diff
@@ -85,6 +86,8 @@ public void anonymousUserCanStartDiscussionWithVideoOnForum() {
   }
 
   @Test(groups = {"ForumBoardTests_006", "ForumBoardTests", "Forum"})
+  @RelatedIssue(issueID = "MAIN-7213", comment = "the failure is caused by the environment instability. " +
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/forumtests/ForumBoardTests.java]: instead of concatenation you can move second argument to new line

**Comment by justnpT** [comment]
@Archbold  Thanks that's correct I've improved the readibility. Back to you 👍

**Pull request merged**